lib/mongoHandler.js

```diff
@@ -48,13 +51,43 @@ MongoStore._getRelationshipAttributeNames = function(attributes) {
 };
 
 
+MongoStore._splitFilterElement = function(filterElementStr) {
```

Comment:
We don't have any unit tests (we should add them at some point!) at the moment, and we're relying on the `jsonapi-server` tests for coverage. Adding unit tests and setting up the unit testing infrastructure is out of the scope of this PR though.